package.json

```diff
@@ -15,6 +15,7 @@
     "@fortawesome/free-regular-svg-icons": "^6.4.2",
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@next/eslint-plugin-next": "^14.0.2",
     "dotenv": "^16.3.1",
     "mongodb": "^6.1.0",
     "next": "^13.5.5",
```

Comment:
Please re-install this package as a dev dependency.
`npm install --save-dev @next/eslint-plugin-next`
Eslint and all associated packages should always be installed as dev dependencies.

Reply:
Please comment when you have address an issue. I can see that the file has been updated. Thank you!!